Page accueil

Coumbadiankha opened the pull request.

Coumbadiankha committed:
Création accueil.dart

Ajouts des imports

Coumbadiankha committed:
Create utils.dart

Coumbadiankha committed:
Update utils.dart

Coumbadiankha committed:
Update utils.dart

Add class MyCustomScrollBehavior

Coumbadiankha committed:
Update utils.dart

Add TextStyle

Coumbadiankha committed:
Update utils.dart

Add try catch

Coumbadiankha committed:
Update accueil.dart

Add Run Debugg

Coumbadiankha committed:
Update main.dart

Add imports

Coumbadiankha committed:
update accueil.dart

mistake

Coumbadiankha committed:
Update main.dart

Coumbadiankha committed:
Update pubspec.yaml

Add assets images

Coumbadiankha committed:
Update

Coumbadiankha committed:
Update

Correction

Coumbadiankha committed:
Update dependencies

Coumbadiankha committed:
Update assets

Add images

Coumbadiankha committed:
Update

Coumbadiankha committed:
Update Dependencies

Coumbadiankha committed:
Update dependencies

Coumbadiankha committed:
Update main.dart

Coumbadiankha committed:
Update

Coumbadiankha committed:
Update

Coumbadiankha committed:
Update assets

Coumbadiankha committed:
Update accueil

Coumbadiankha committed:
Update page_accueil

Coumbadiankha committed:
Update carousel

Coumbadiankha committed:
Update page_accueil

taniaolivia reviewed (changes requested):
Les deux grands titres du carousel des conférences et des stands, il faut les retravailler parce qu'ils sont trop tirés

taniaolivia reviewed (changes requested):
Mettre tous les noms du fichier et des assets en anglais

Mat0108 reviewed (changes requested):
Request : 
- Nomme les images et les filtrer
- Nommer les ficher dart en anglais
- Retirer les utils.dart

Coumbadiankha committed:
Update Home

Coumbadiankha committed:
Update page_accueil

- Correction du nom des images
- Correction du nom des fichiers nettoyage du code.

Commit:
fix image doesn't load

Coumbadiankha committed:
Update carousel

Coumbadiankha committed:
Update home.dart

Coumbadiankha committed:
Update color.dart

deplacer vers components

Coumbadiankha committed:
update color.dart
